group_vars/bibdata.yml

```diff
@@ -1,4 +1,34 @@
 ---
+datadog_config:
+  tags: "application:bibdata, environment:production, type:webserverj"
```

Comment:
I noticed some apps say `type:webserver` and some say `type:webserverj` — is this a typo? Or actually a different type?

Reply:
@escowles Fixed.